doc_src/index.hdr.in

```diff
@@ -849,12 +849,14 @@ The user can change the settings of `fish` by changing the values of certain var
 
 - `FISH_VERSION`, the version of the currently running fish
 
+- `SHLVL`, the level of nesting of shells
+
+These two variables are not read-only and can be set by the user. They will be used if the `stty` rows and cols values are zero (which can happen on a serial console). Fish checks that the values re reasonable which means cols >= 20 and rows >= 2. Note that fish will also modify these vars to match what it believes to be the size of the terminal. This means that the terminal size reported by the kernel has precedence over these variables.
```

Comment:
This is a bit impenetrable. Why not move LINES and COLUMNS to the previous section, with text along the lines of:

"- `LINES` and `COLUMNS`, the current size of the terminal in height and width. These values are only used by fish if the operating system does not report the size of the terminal. They are updated when the window size changes."